code/modules/clothing/suits/bio.dm

```diff
@@ -6,7 +6,7 @@
 	worn_icon = 'icons/mob/clothing/head/bio.dmi'
 	icon_state = "bio"
 	inhand_icon_state = "bio_hood"
-	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | SNUG_FIT | PLASMAMAN_HELMET_EXEMPT
+	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | SNUG_FIT | PLASMAMAN_HELMET_EXEMPT | MASKINTERNALS
```

Comment:
It is a clothing flag which can be used on more than just masks, however I see why it might be confusing to re-use it. Would you like me to add a `HEADINTERNALS` define instead?

Reply:
yes

Reply:
Fixed by commit d04a8412f562beec3ec01a88e7c51f305d90c5d5 and commit 3d3fc239644f88ab70d29e685536e8afd34f316d